rest/authorization-server/src/test/java/org/eclipse/sw360/rest/authserver/GrantTypeClientCredentialsCustomHeaderAuthTest.java

```diff
@@ -24,6 +25,7 @@
  * A POST request for an access token with grant type 'client_credentials' and
  * custom auth header should NOT be possible.
  */
+@Ignore
```

Comment:
Please add a reason for ignoring the testcases with `@Ignore("reason here")` syntax. It will make it easier to understand when this can be re-enabled.